packages/component-header/src/components/HeaderMain/NavbarContainer/index.styles.js

```diff
@@ -23,7 +23,6 @@ const Wrapper = styled.nav`
       flex-direction: column;
       justify-content: flex-start;
       overflow-y: auto;
-      min-height: calc(100vh - 277px);
```

Comment:
Keep this code, we want the nav to take over the entire screen if there is content inside "content-container"

Reply:
@scott-williams-az  got it, just to confirm. If there is one link we want it to take up the entire screen area. But if no links it should not take up the entire screen height. As shown in the [design docs](https://xd.adobe.com/view/56f6cb78-9af5-4b12-b4ce-ef319f71113f-03a5/screen/02335e90-65c1-4245-8fc7-71b70322c4ff/)

Reply:
Correct

Reply:
completed